packages/wpcom-checkout/src/transformations.ts

```diff
@@ -42,19 +42,6 @@ export function getCouponLineItemFromCart( responseCart: ResponseCart ): LineIte
 	};
 }
 
-export function getSubtotalLineItemFromCart( responseCart: ResponseCart ): LineItemType {
-	return {
-		id: 'subtotal',
-		type: 'subtotal',
-		// translators: The label of the subtotal line item in checkout
```

Comment:
Not sure if it's worth keeping this translator comment in place in the new code?

Reply:
I imagine (?) it's not necessary since the string has already been translated.